lib/interface_quda.cpp

```diff
@@ -5311,3 +5912,188 @@ void gaugeObservablesQuda(QudaGaugeObservableParam *param)
 
   gaugeObservables(*gauge, *param);
 }
+
+#if 0
```

Comment:
What purpose does this code serve (given it's all commented out)?

Reply:
Delete this code

Reply:
the code was deleted in some commit